README.md

```diff
@@ -138,10 +139,36 @@ For example, to use the Infracost CLI with the latest official Atlantis image, a
 
 To test, send a new pull request to change something in Terraform that costs money; a comment should be posted on the pull request by Atlantis. Expand the Show Output section, at the bottom of which you should see the Infracost output.
 
-## Contributing
+# Project Examples
+
+To help you get up and running with Infracost and Atlantis as quick as possible, we've compiled a list of commonly used scenarios. 
+
+* [Single Project](./examples/single_project/README.md)
+
+# Contributing
 
 Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change.
 
-## License
+## Running Infracost Atlantis Docker image locally
```

Comment:
Should this stuff should go into CONTRIBUTING.md? so the readme is focused on users.